device-types/HPE/Aruba-2920-48G-PoEP.yaml

```diff
@@ -5,11 +5,13 @@ slug: hpe-aruba-2920-48g-poep
 part_number: J9729A
 u_height: 1
 is_full_depth: false
+weight: 5700
+weight_unit: g
 power-ports:
   - name: PS1
     type: iec-60320-c14
-    maximum_draw: 575
-    allocated_draw: 487
+    maximum_draw: 487
+    allocated_draw: 46
```

Comment:
It appears that the PSU is hot swappable. Thus it should be modelled under `module-bays`, and then the associated PSU module should be modelled in the module-types folder if they do not already exist there

If I am wrong about this please let me know and mark this as resolved